scripts/src/components/organisation-form.js

```diff
@@ -98,12 +113,10 @@ var CreateOrg = React.createClass({
     if ('star_rating' in data) {
       data['star_rating'] = parseInt(data['star_rating'] || 0);
     }
-    data['reference_links'] = this.state.referenceLinks;
-    if (id) {
-      actions.updateOrganisation.push(_.extend(data, { organisationId: id }));
-    } else {
-      actions.createOrganisation.push(data);
-    }
+
+    data['reference_links'] = this._prepareReferenceLinks();
+    if (id) { actions.updateOrganisation.push(_.extend(data, { organisationId: id })); }
```

Comment:
I'm not a huge fan of inlining if statements like this, is there a particular code convention that says we should be doing it this way?